angelssmelting/changelog.txt

```diff
@@ -1,4 +1,15 @@
 ---------------------------------------------------------------------------------------------------
+Version: 0.6.2
+Date: ??
+  Changes:
```

Comment:
Add a Change entry that you moved `"angels-coil-glass-fiber"` from glass 3 to 1

Reply:
Resolved in https://github.com/Arch666Angel/mods/pull/89/commits/efdd289df39a662c5676ec8ef8fef515d91ee952